Makefile

```diff
@@ -20,6 +20,8 @@ MAIL_SERVER ?= "smtp.gmail.com"
 MAIL_ADDRESS ?= "omeka@example.com"
 MAIL_PASSWORD ?= replace_me
 
+.PHONEY: init-app
```

Comment:
@nomadicoder I am specifically asking about this:
.PHONEY: init-app

I don't see that being used anywhere, so I don't understand the purpose.

Reply:
See above change.  I resolve it just before your last comment